tools/package-client.js

```diff
@@ -463,6 +566,38 @@ exports.publishPackage = function (options) {
     }
   }
 
+  // Check that our documentation exists (or we know that it doesn't) and has
+  // been filled out.
+  var readmeInfo;
+  buildmessage.enterJob("processing docs", function () {
+    readmeInfo = packageSource.processReadme();
+  });
+  if (buildmessage.jobHasMessages())
+    return;
+  if (readmeInfo && (readmeInfo.hash === files.blankHash)) {
+    buildmessage.error(
+      "Your documentation file is blank, so users may have trouble figuring " +
+      "out how to use your package. Please fill it out, or " +
+      "set 'documentation: null' in your Package.describe");
+  }
+
+  if (readmeInfo &&
+      readmeInfo.excerpt.length > 1500) {
```

Comment:
Yeah, sorry about that. I will ping you explicitly when ready for review, I guess.

Do you want me to remove the check altogether, or is just moving it to later sufficient?